src/azure-cli-testsdk/azure/cli/testsdk/preparers.py

```diff
@@ -46,7 +46,7 @@ def live_only_execute(self, cli_ctx, command, expect_failure=False):
 class ResourceGroupPreparer(NoTrafficRecordingPreparer, SingleValueReplacer):
     def __init__(self, name_prefix='clitest.rg',
                  parameter_name='resource_group',
-                 parameter_name_for_location='resource_group_location', location='westus',
+                 parameter_name_for_location='resource_group_location', location='eastus',
```

Comment:
nit: Should we set this to `unitedstates`

Reply:
Why change this?

Reply:
the location 'westus' no longer works in iotc, we need to change it to something else, for example, 'eastus'
As you suggested, I passed location='eastus' to the decorator.